packages/cactus-plugin-htlc-eth-besu-erc20/src/main/json/openapi.json

```diff
@@ -0,0 +1,354 @@
+                        "nullable": false
+                    }
+                }
+            }
+        },
+        "responses": {
+            "getStatusResponse": {
+                "description": "Array of status",
+                "content": {
+                  "application/json": {
+                    "schema": {
+                      "type": "array",
+                      "items": {
+                        "type": "integer",
+                        "format": "uint256"
+                      }
+                    }
+                  }
+                }
+              },
+            "getSingleStatusResponse": {
+                "description": "Status",
+                "content": {
+                  "application/json": {
+                    "schema": {
+                      "type": "integer",
+                      "format": "uint256"
+                    }
+                  }
+                }
+            }
+        }
+    },
+    "paths": {
+        "/api/v1/plugins/@hyperledger/cactus-plugin-htlc-eth-besu-erc20/withdraw": {
+            "post": {
+                "operationId": "withdraw",
+                "summary": "Withdraw a hashtimelock contract",
+                "parameters": [],
+                "requestBody": {
+                    "content": {
+                        "application/json": {
+                            "schema": {
+                                "$ref": "#/components/schemas/WithdrawRequest"
+                            }
+                        }
+                    }
+                },
+                "responses": {
+                    "200": {
+                        "description": "OK",
+                        "content": {
+                            "application/json": {
+                                "schema": {
+                                    "$ref": "#/components/schemas/WithdrawResponse"
+                                }
+                            }
+                        }
+                    }
+                }
+            }
+        },
+        "/api/v1/plugins/@hyperledger/cactus-plugin-htlc-eth-besu-erc20/get-status": {
+            "get": {
+                "operationId": "getStatus",
+                "summary": "Get array of status of a hashtimelock contract, the status are ( 0 - Invalid, 1 - Active, 2 - Refunded, 3 - Withdrawn, 4 - Expired)",
+                "parameters": [
+                  {
+                    "name": "ids",
+                    "in": "query",
+                    "required": true,
+                    "schema": {
+                      "type": "array",
+                      "items": {
+                        "type": "string",
+                        "format": "bytes32"
+                      }
+                    }
+                  }
+                ],
+                "responses": {
+                  "200": {
+                    "description": "OK",
+                    "$ref": "#/components/responses/getStatusResponse"
```

Comment:
@jordigiam Please have the schema types start with uppercase

Reply:
Done